src/rgw/rgw_rest.cc

```diff
@@ -574,6 +574,7 @@ void abort_early(struct req_state *s, RGWOp *op, int err_no)
 
 void dump_continue(struct req_state *s)
 {
+  perfcounter->inc(l_rgw_http_status_1xx);
```

Comment:
Why is the 1xx counter incremented here too? Is it a different code path than the one in `rgw_main.cc`?